README.md

```diff
@@ -64,6 +64,8 @@ See the [community](#community) section for ways to contact us.
 Please raise an issue
 [in this repository](https://github.com/kata-containers/kata-containers/issues).
 
+__If you are reporting a security issue, please follow the [vulnerability reporting process](https://github.com/kata-containers/community/blob/master/VMT/VMT.md#reception)__.
```

Comment:
Could you make this a "note" as documented in https://github.com/kata-containers/kata-containers/blob/2.0-dev/docs/Documentation-Requirements.md#notes?

That will make it "stand out" a lot more than simply making it bold. Currently, it doesn't stand out at all imho since GH renders headings in bold too so this kind of bleeds into the heading below ("**Kata Containers 1.x versions**").

Reply:
It's best to direct researchers / folks who want to submit a security issue to a page that explains how the advisory should be reported. I think linking [here](https://github.com/kata-containers/community#reporting-vulnerabilities) may be a better fit. The VMT document is rather long and is more about how a vulnerability is handled internally by Kata maintainers, and doesn't include some of the instructions available [here](https://github.com/kata-containers/community#reporting-vulnerabilities).

Reply:
Alternatively you could add a new large title to the VMT doc dubbed "Reporting Vulnerabilities" and link there.

Reply:
Sounds good. Thanks @yuvalavra and @jodh-intel 
AFK this week, but will update the PR soon!